### ISSUE TITLE
Streams API notification

### ISSUE THREAD

**Issue by maddoxd** [aws/amazon-connect-streams#93]
Hi

We have noticed that our softphone using Streams API does not pop chrome notifications like standard CCP does.  Is this something that can easily be turned on? or do we have to hand code it?

**Comment by jagadeeshaby** [comment]
Streams provide the useful interface for handling notifications which is `connect.core.getNotificationManager() `

This is how we can use this. 

```
      const notificationManager = connect.core.getNotificationManager();
        notificationManager.show("Title", {
          body: "Message Body",
          clicked: function () {
            // some actions. 
          }
        });
```

## Usage example for incoming call (Have not tested - please do validate)

```
connect.contact(function(contact){  
  contact.onConnecting(function() {
    if (contact.isInbound()) {
      const incomingPhoneNumber  = contact.getInitialConnection().getAddress().stripPhoneNumber();     
      connect.ifMaster("NOTIFICATIONS", function () {
        const notificationManager = connect.core.getNotificationManager();
        notificationManager.show("Call from" + incomingPhoneNumber, {
          body: "Click here to open the contact control panel now.",
          clicked: function () {
            window.focus();
            this.close();
          }
        });
      });
    }
  });
});
```

**Comment by jagadeeshaby** [comment]
Please feel free to reopen the issue if you need more help

**Comment by Swethamr402** [comment]
Hi 
I used the same code but notificationManager.show - this line is not executing. Is there any configuration do we need to do for chrome browser.

**Comment by mdebusschere2** [comment]
I tried to get this too work multiple ways.  

The const notificationManager = connect.core.getNotificationManager();  actually helped restore notifications which we didn't see in our app.

Using the .show(), puts the request behind the built in notification..  I tried clearing the NM queue beforehand
but the default notification always appears first, if you click on first notification, then the custom built one appears next in our testing.
